crates/ruff_benchmark/benches/red_knot.rs

```diff
@@ -27,10 +27,10 @@ static EXPECTED_DIAGNOSTICS: &[&str] = &[
     "/src/tomllib/_parser.py:7:29: Module `collections.abc` has no member `Iterable`",
     // We don't support terminal statements in control flow yet:
     "/src/tomllib/_parser.py:66:18: Name `s` used when possibly not defined",
+    "/src/tomllib/_parser.py:108:17: Conflicting declared types for `second_char`: Unknown, str | None",
     "/src/tomllib/_parser.py:98:12: Name `char` used when possibly not defined",
     "/src/tomllib/_parser.py:101:12: Name `char` used when possibly not defined",
     "/src/tomllib/_parser.py:104:14: Name `char` used when possibly not defined",
-    "/src/tomllib/_parser.py:108:17: Conflicting declared types for `second_char`: Unknown, str | None",
     "/src/tomllib/_parser.py:115:14: Name `char` used when possibly not defined",
```

Comment:
I couldn't figure out why the order of the diagnostics changed here as nothing obvious stands out to me.

Reply:
This is resolved in https://github.com/astral-sh/ruff/pull/13978/commits/8d3a2ff5ee818405fbd7f2c28ed37d6b58a741d0